packages/quick_actions/quick_actions_ios/ios/Classes/ShortcutItemParser.swift

```diff
@@ -12,26 +12,24 @@ protocol ShortcutItemParser {
   /// - Parameter items an array of raw shortcut items to be parsed.
   /// - Returns an array of parsed shortcut items to be set.
   ///
-  func parseShortcutItems(_ items: [[String: Any]]) -> [UIApplicationShortcutItem]
+  func parseShortcutItems(_ items: [ShortcutItemMessage]) -> [UIApplicationShortcutItem]
```

Comment:
can you add a TODO to remove this parser since the parsing logic is handled by pigeon now.

Reply:
It isn't entirely, since this converts the `ShortcutItem` to a `UIApplicationShortcutItem`.

Reply:
I removed it instead. Probably worth one more look through